.github/workflows/python-ci.yml

```diff
@@ -5,32 +5,37 @@ on: push
 jobs:
   test:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.7, 3.8]
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+
     - name: Install dependencies
-      env:
-        PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
       run: |
         sudo apt-get update
         sudo apt-get install libhdf5-serial-dev netcdf-bin libnetcdf-dev
         sudo apt-get install cdo
-        pip install -U pip pytest
+        pip install -U pip pipenv
+
+    - name: Install python packages
+      run: |
+        pipenv lock --requirements > requirements.txt
+        pipenv lock --requirements --dev >> requirements.txt
```

Comment:
Can you explain why you are doing this and not just using `pipenv` to install? Maybe in a comment, unless my confusion is very naive?

Reply:
This was a way around some versioning issues. It will be updated with the next set of changes in this PR.